crates/rmf_site_editor/src/site_asset_io.rs

```diff
@@ -160,13 +161,21 @@ async fn fetch_asset<'a>(
     Ok(Box::new(VecReader::new(bytes)))
 }
 
-fn get_path_from_env() -> Result<PathBuf, env::VarError> {
+fn get_path_from_env() -> Result<Vec<PathBuf>, env::VarError> {
     let var = env::var(MODEL_ENVIRONMENT_VARIABLE)?;
-    let path = PathBuf::from(var);
+    let split: SmallVec<[&str; 8]> = var.split(':').collect();
```

Comment:
We're going to run into some issues with this in Windows since path names tend to start with `C:\\...`.

What we should really use here is [`std::env::split_paths`](https://doc.rust-lang.org/std/env/fn.split_paths.html).